.github/workflows/monitor.yml

```diff
@@ -0,0 +1,44 @@
+name: Root Monitor
+
+# Execute this as a daily cron job.
+on:
+  # Enable cron for re-signing snapshot and timestamp every week
+  schedule:
+    - cron: '0 0 */1 * *' # every day
+  workflow_dispatch:
+
+jobs:
+  root-probe:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.7
+      - name: Install verify
+        run: go install github.com/sigstore/root-signing/...@latest
+
+      # Check expiration in less than 3 days
+      - name: Check expiration
+        run: |
+          export EXPIRY=$(date -d '+3 days' '+%Y/%m/%d')
+          verify repository --gcs --repository sigstore-tuf-root --root ceremony/2021-06-18/repository/root.json --valid-until ${EXPIRY}
```

Comment:
I like how you've done the heavy lifting in the root-signing CLI tools here, nice!